packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -331,13 +345,23 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
             >
               <StartCallButton
                 className={mobileWithPreview ? startCallButtonStyleMobile : undefined}
-                onClick={startCallHandler}
+                onClick={startCall}
                 disabled={disableStartCallButton}
               />
             </Stack>
           </Stack>
         </Stack>
-        <SidePane mobileView={props.mobileView} updateSidePaneRenderer={props.updateSidePaneRenderer} />
+        <Panel
```

Comment:
Are we switching back to fluent panel for Config screen? In case we would like to switch content for side pane in future, we will have to introduce new panel for that.

Reply:
Yeah the figma's show it above the content so this was simplest just to host the side pane in a panel.
Sorry not following the second part of the comment - what do you mean by "we will have to introduce new panel for that"?

Reply:
If we plan on adding another panel content for example for sound effects, would we need to create a new panel for sound effects or we would be able to override content in the same panel

Reply:
Nope same panel, this just adjusts how it is hosted in the configuration pane, the content is tangential to that

Reply:
(on re-looking, the isopen param would need to be updated though but I'll leave as is for now)

Reply:
Sounds good